test/src/main/java/org/jvnet/hudson/test/HudsonTestCase.java

```diff
@@ -501,7 +506,19 @@ protected Hudson newHudson() throws Exception {
         File home = homeLoader.allocate();
         for (Runner r : recipes)
             r.decorateHome(this,home);
-        return new Hudson(home, createWebServer(), useLocalPluginManager ? null : pluginManager);
+        if (useLocalPluginManager) {
+            pluginManager = new LocalPluginManager(home) {
+                @Override
+                protected Collection<String> loadBundledPlugins() {
+                    // Overriding so we can force loading of the detached plugins for testing
```

Comment:
It probably could but that wouldn't cover tests outside Jenkins core that are depending on this. I think there was only one test using it inside Jenkins core.